sdk/recoveryservices-backup/Microsoft.Azure.Management.RecoveryServices.Backup/src/recoveryservicesbackup/Generated/Models/AzureVmWorkloadProtectedItemExtendedInfo.cs

```diff
@@ -32,16 +32,25 @@ public AzureVmWorkloadProtectedItemExtendedInfo()
         /// AzureVmWorkloadProtectedItemExtendedInfo class.
         /// </summary>
         /// <param name="oldestRecoveryPoint">The oldest backup copy available
-        /// for this backup item.</param>
+        /// for this backup item across all tiers.</param>
+        /// <param name="oldestRecoveryPointInVault">The oldest backup copy
+        /// available for this backup item in vault tier</param>
+        /// <param name="oldestRecoveryPointInArchive">The oldest backup copy
+        /// available for this backup item in archive tier</param>
+        /// <param name="newestRecoveryPointInArchive">The latest backup copy
+        /// available for this backup item in archive tier</param>
         /// <param name="recoveryPointCount">Number of backup copies available
         /// for this backup item.</param>
         /// <param name="policyState">Indicates consistency of policy object
         /// and policy applied to this backup item.</param>
         /// <param name="recoveryModel">Indicates consistency of policy object
         /// and policy applied to this backup item.</param>
-        public AzureVmWorkloadProtectedItemExtendedInfo(System.DateTime? oldestRecoveryPoint = default(System.DateTime?), int? recoveryPointCount = default(int?), string policyState = default(string), string recoveryModel = default(string))
+        public AzureVmWorkloadProtectedItemExtendedInfo(System.DateTime? oldestRecoveryPoint = default(System.DateTime?), System.DateTime? oldestRecoveryPointInVault = default(System.DateTime?), System.DateTime? oldestRecoveryPointInArchive = default(System.DateTime?), System.DateTime? newestRecoveryPointInArchive = default(System.DateTime?), int? recoveryPointCount = default(int?), string policyState = default(string), string recoveryModel = default(string))
```

Comment:
breaking change

Reply:
Added customization files for breaking changes